app/views/katello/subscription_mailer/subscription_expiry.html.erb

```diff
@@ -40,6 +40,7 @@
         </tr>
       <% end %>
     </table>
+  <%= link_to _("Report"), report_url %>
```

Comment:
I think this should be outside the `<div>`, maybe even with a `<br />` to separate it.  (See picture)

![email_with_report_link](https://user-images.githubusercontent.com/22042343/80648764-be16eb80-8a3e-11ea-97f4-55fbd65e2b42.png)